server/nuxeo-nxr-server/src/main/resources/templates/common-base/nxserver/config/cors-config.xml.nxftl

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.platform.ui.web.cors.config">
+
+<#if (nuxeo.cors.urls)?has_content>
+  <extension target="org.nuxeo.ecm.platform.web.common.requestcontroller.service.RequestControllerService" point="corsConfig">
+      <corsConfig name="cors" allowOrigin="${nuxeo.cors.urls}" allowSubdomains="${nuxeo.cors.allowSubdomains}" supportedMethods="${nuxeo.cors.supportedMethods}">
+        <pattern>/.*</pattern>
```

Comment:
Do we want to make the pattern configurable?

Reply:
I have observed some slight variations in the pattern.  While I don't think the differences have much purpose, it seems a good idea to make this configurable (just in case).  @npeacock88 Can you please make the pattern configurable with the default pattern nuxeo.cors.pattern=/.*